jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -436,6 +436,7 @@ private void commitSession(Session session)
                 }
                 catch (IllegalStateException | IllegalArgumentException e)
                 {
+                    LOG.warn(e.getLocalizedMessage());
```

Comment:
Normally we've just logged e.toString(). Is this better?

Reply:
toString() calls getLocalizedMessage() after prepending the classname. I'll change it to toString().